codegen/test_data/endpoints/bar_bar_method_argnotstruct.gogen

```diff
@@ -26,14 +26,18 @@ func HandleArgNotStructRequest(
 	}
 
 	headers := map[string]string{}
+	// TODO(sindelar): Add optional headers in addition to required.
+	for _, k := range []string(nil) {
```

Comment:
It is strange to have generated code ranging over nil. Can we make template copy headers only if there are headers in request?